codecparsers/h264Parser_unittest.cpp

```diff
@@ -173,7 +172,7 @@ namespace H264 {
         void checkH264SliceHeader(Parser& parser, NalUnit* nalu)
         {
             SharedPtr<SliceHeader> slice(new SliceHeader);
-            memset(slice.get(), 0, sizeof(SliceHeader));
+            *slice = SliceHeader();
```

Comment:
why we need this if SliceHeader has a constructor. the member already inited by "new SliceHeader"

Reply:
@xuguangxin , I have removed the line of code.